rpcserver.go

```diff
@@ -1091,8 +1091,10 @@ func (s *rpcServer) RecoverAccounts(ctx context.Context,
 	var recoveredAccounts []*account.Account
 	var err error
 
+	target := req.AccountTarget
+
 	if !req.FullClient {
-		recoveredAccounts, err = s.serverAssistedRecovery(ctx)
+		recoveredAccounts, err = s.serverAssistedRecovery(ctx, target)
```

Comment:
According to the commit message this should now fall back to `DefaultAccountKeyWindow`. Or only if `target == 0`?

Reply:
Comment & code updated.